make/build.js

```diff
@@ -39,17 +40,29 @@ module.exports = function build(settings, task) {
 
     var version = getversion.factory("build/js/Version.js", "var version");
 
-    task("plain", [], function() {
+    task("cs2js", [], function() {
+        this.input("tools/cs2js.js");
+        var cssrc = this.input(src.cssrc);
+        var dst = this.output("build/js/Compiled.js");
+        this.addJob(function() {
+            return require("../tools/cs2js").compileFiles(cssrc)
+                .then(function(jscode) {
+                    return qfs.write(dst, jscode);
+                });
+        });
+    });
+
+    task("plain", ["cs2js"], function() {
         version(this);
         this.concat(src.srcs, "build/js/Cindy.plain.js");
     });
 
-    task("ours", [], function() {
+    task("ours", ["cs2js"], function() {
```

Comment:
Already did that in 7fdc737671c772bfa39ce65e88d1407ab6b4dc0c as discussed in https://github.com/CindyJS/CindyJS/pull/154#issuecomment-173406936.